core-kotlin-modules/core-kotlin-9/src/test/kotlin/com/baeldung/underscoreoperator/UnderScoreOperator.kt

```diff
@@ -0,0 +1,34 @@
+package com.baeldung.underscoreoperator
+
+// Before Kotlin 1.7
+class Box<T>(value: T) {
+    // Some class implementation
+}
+fun main() {
+    val intBox: Box<Int> = Box(42)
+    val anyBox: Box<Any> = Box("Some value")
+}
+
+
+fun main1(args: Array<String>) {
+    val (_, second) = "ignored" to "hello"
+    println(second)
+}
+
+fun mainTwo(args: Array<String>) {
+    val list = listOf("hi")
+    val list2 = list.mapIndexed { _, item -> item }
+    println(list2)
+}
+
+
+inline fun <T> printElementInfo(element: T) {
+}
```

Comment:
Can we still just print the element?

```suggestion
inline fun <T> printElementInfo(element: T) {
    println("Element: $element")
}
```